screens/Home/HomeScreen.tsx

```diff
@@ -76,6 +77,12 @@ export const HomeScreen: React.FC<HomeRouteProps> = props => {
         )}
       </Column>
       {isOpenId4VCIEnabled() && <DownloadFABIcon />}
+      <ErrorMessageOverlay
```

Comment:
Have we removed the overlay from MyVcsTab.tsx, which was used earlier in the downloadcard scenario

Reply:
We have not cleaned up the download card button still so I have not removed this storage error overlay as well from MyVCsTab.tsx

Reply:
okay, can you please add this as a sub task in the same clean up story

Reply:
Noted